nomos-services/data-availability/network/src/lib.rs

```diff
@@ -600,6 +602,32 @@ where
             tracing::error!("No membership found for session {session_id}");
         }
     }
+
+    async fn handle_opinion_session_change(
+        opinion_aggregator: &mut OpinionAggregator<Membership, Arc<StorageAdapter>>,
+        current_membership: Membership,
+    ) {
+        match opinion_aggregator
+            .handle_session_change(current_membership)
+            .await
+        {
+            Ok(Some(opinions)) => {
+                tracing::debug!(
+                    "Processing opinions - session_id: {}, new_opinions: {}, old_opinions: {}",
+                    opinions.session_id,
+                    opinions.new_opinions,
+                    opinions.old_opinions
+                );
+                // todo: sdp_adapter.process_opinions(opinions).await;
+            }
+            Ok(None) => {
+                tracing::debug!("No opinions generated yet");
```

Comment:
Shouldn't (when sdp.process_opinions is implemented) this send a bitvec of opinions with all zeroes?